src/applications/personalization/profile360/actions/paymentInformation.js

```diff
@@ -20,7 +20,7 @@ export const PAYMENT_INFORMATION_SAVE_SUCCEEDED =
 export const PAYMENT_INFORMATION_SAVE_FAILED =
   'PAYMENT_INFORMATION_SAVE_FAILED';
 
-export function fetchPaymentInformation() {
+export function fetchPaymentInformation(recordEvent = recordAnalyticsEvent) {
```

Comment:
Using dependency injection here so I test that the `recordEvent` helper is called correctly. I'm open to other ideas to make the calls to `recordEvent` testable.

Reply:
I sometimes add an underscore to the import to help draw the distinction -

```
export function fetchPaymentInformation(recordAnalyticsEvent) = _recordAnalyticsEvent) {
```

Reply:
Interesting idea